packages/fluentui/react-northstar/src/themes/teams-dark/components/Attachment/attachmentVariables.ts

```diff
@@ -1,13 +1 @@
-import { AttachmentVariables } from '../../../teams/components/Attachment/attachmentVariables';
-
-export default (siteVariables: any): Partial<AttachmentVariables> => ({
-  textColor: siteVariables.colors.white,
-  textColorHover: siteVariables.colors.white,
-
-  backgroundColor: siteVariables.colors.grey[550],
-  backgroundColorHover: siteVariables.colors.grey[500],
-  borderColor: siteVariables.colors.onyx[700],
-  boxShadow: siteVariables.shadowLevel1,
-
-  actionColor: siteVariables.colors.white,
-});
+export { default } from '../../../teams/components/Attachment/attachmentVariables';
```

Comment:
remove these file if no overrides for variables are required. also remove the export from componentVariables under teams-dark